### PR TITLE
added doc page on public url

### LINKED ISSUE #3232: Add documentation on importance of the public facing url
It can never change, is used to create did web and revocation lists.

### PR DESCRIPTION
**Pull request by woutslakhorst**
closes #3232 

top item for deployment

### CODE REVIEW

File: docs/pages/deployment/domain.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Changing the domain name
+************************
+
+Changing the domain name of a Nuts node is basically the same as setting up a new node on a new domain.
+All DIDs, VerifiableCredentials, Revocations, and other data is no longer usable.
+You will not be able to revoke credentials. This means that all credentials issued by the old domain can no longer be trusted.
+Actions should be taken to remove any trust that might have been established in your old identity.
+
+As an issuer, you'll have to run the new domain side-by-side with the old domain for at least a year.
```
Comment (gerardsn) [inline docs/pages/deployment/domain.rst:44]: why? Is this an estimate how long it takes to transfer the trust? (how long it takes for verifiers to accept the new issuer)

File: docs/pages/deployment/domain.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+.. code-block:: yaml
+
+    url: https://example.com
+
+The domain name is used in the following ways:
+
+- It provides some information about the owner of DIDs and is part of the client_id in OAuth flows.
+  Other parties can use the domain name to identify your node.
+- It's part of DIDs you create.
```
Comment (gerardsn) [inline docs/pages/deployment/domain.rst:16]:
```suggestion
- It's part of the DIDs the node creates for you when you create a new subject.
```

File: docs/pages/deployment/domain.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- You must own the domain name.
+- The domain name should be stable.
+  It should not change.
+- It should use a TLD that allows for retention of the domain name.
+  For example, a .com domain name can be blocked for a period of time after it's no longer registered.
+  This will prevent the next owner from using it.
+- The domain name should be human readable.
+  Sub-domains from cloud providers are not recommended since they don't provide information about the owner.
+- There should be a security.txt and robots.txt file at the root of the domain.
+  This is a best practice for security and privacy.
```
Comment (gerardsn) [inline docs/pages/deployment/domain.rst:34]: this is part of security, not the domain name?

**Review by gerardsn — APPROVED** [review]

File: docs/pages/deployment/domain.rst (hunk 1 above)
Comment (woutslakhorst) [reply docs/pages/deployment/domain.rst:44]: added note

File: docs/pages/deployment/domain.rst (hunk 3 above)
Comment (woutslakhorst) [reply docs/pages/deployment/domain.rst:34]: tells people to take the domain seriously.

**Pull request merged**